Makefile

```diff
@@ -217,7 +221,10 @@ deploy-ingress-nginx:
 		kubectl --namespace ingress-nginx describe pod -l app.kubernetes.io/component=controller; \
 	done
 
-deploy-apiserver: HELM_FLAGS ?=--set=image.tag=$(GIT_VERSION) --set=image.pullPolicy=IfNotPresent --set=konk.name=${KONK_NAME}
+ifndef KONK_NAME
+deploy-apiserver: CREATE_KONK ?= true
+endif
+deploy-apiserver: HELM_FLAGS ?=--set=image.tag=$(GIT_VERSION) --set=image.pullPolicy=IfNotPresent --set=konk.create=${CREATE_KONK} --set=konk.name=${KONK_NAME}
```

Comment:
👍

Reply:
I could pull this out into its own PR too, if you want it sooner.

Reply:
#111